.github/workflows/release.yaml

```diff
@@ -103,6 +108,7 @@ jobs:
           files: |
             release-artifacts/install.yaml
             release-artifacts/quickstart.yaml
+            release-artifacts/envoy-gateway-crds.yaml
```

Comment:
How is this come?

Reply:
I had screwed up my different branches when I fixed this. The file now gets generated on line 78 and gets published here.

Reply:
This is generated from the make command now. And it publishes the EG CRD to make it easy users of the current helm chart to apply the new CRD version.